Ciac 16336 checkpointv2 add new command phase3 (#43948)
demisto/content: Packs/CheckpointFirewall/ReleaseNotes/2_3_38.md

#### Integrations

##### CheckPoint Firewall v2

- Updated the Docker image to: *demisto/python3:3.12.13.8428455*.

> - Added support for **checkpoint-access-section-delete** command that deletes the specified access section.

Reviewer: ```suggestion
- Added support for the following commands:  
  - ***checkpoint-access-section-delete***, that deletes the specified access section.
```

> - Added support for **checkpoint-access-section-get** command that shows an existing access section using object name or uid.

Reviewer: ```suggestion
  - ***checkpoint-access-section-get***, that shows an existing access section using object name or UID.
```

> - Added support for **checkpoint-access-section-add** command that adds a new access section.

Reviewer: ```suggestion
  - ***checkpoint-access-section-add***, that adds a new access section.
```

> - Added support for **checkpoint-service-group-add** command that adds a new service group object.

Reviewer: ```suggestion
  - ***checkpoint-service-group-add*** that adds a new service group object.
```

> - Added support for **checkpoint-service-group-get** command that shows an existing service group object using object name or uid.

Reviewer: ```suggestion
  - ***checkpoint-service-group-get***, that shows an existing service group object using object name or UID.
```

> - Added support for **checkpoint-service-group-update** command that updates the specified service group object.

Reviewer: ```suggestion
  - ***checkpoint-service-group-update***, that updates the specified service group object.
```

> - Added support for **checkpoint-service-group-list** command that gets a list of all service group objects.

Reviewer: ```suggestion
  - ***checkpoint-service-group-list***, that gets a list of all service group objects.
```

> - Added support for **checkpoint-service-group-delete** command that deletes the specified service group object.

Reviewer: ```suggestion
  - ***checkpoint-service-group-delete***, that deletes the specified service group object.
```

> - Added support for **checkpoint-access-section-update** command that updates the specified access section.

Reviewer: ```suggestion
  - ***checkpoint-access-section-update***, that updates the specified access section.
```

> - Added support for **checkpoint-service-group-clone** command that clones an existing service group object.

Reviewer: ```suggestion
  - ***checkpoint-service-group-clone***, that clones an existing service group object.
```
